docs/guides/typescript.md

```diff
@@ -267,31 +273,30 @@ By following this approach, you can ensure that your code handles dynamic `repla
 ### Middleware that doesn't change the store type
 
 ```ts
-import { create, State, StateCreator, StoreMutatorIdentifier } from 'zustand'
```

Comment:
A working example can be found here: https://stackblitz.com/edit/vitejs-vite-mbyits?file=src%2FApp.tsx